tests/fft.c

```diff
@@ -0,0 +1,69 @@
+#define FFT_WRITE_LANE  0x2000
```

Comment:
Can you add a general comment indicating that this should be used with the FFT generator config